src/components/ItemsList.js

```diff
@@ -43,43 +44,55 @@ const ItemsList = () => {
     history.push('/additem');
   };
 
+  const listHasAtLeastOneItem = shoppingList && shoppingList[0];
+  const listHasNoItems = shoppingList && !shoppingList.length;
 
   return (
     <div className="items-list">
       <h1>Your Shopping List</h1>
       {loading && <p>Loading...</p>}
       {error && <p>An error has occured...</p>}
-      {shoppingList && !shoppingList.length && (
-            <div className="add-item">
-              <p>You haven't created a shopping list yet...</p>
-              <button type="submit" onClick={handleRedirect}>
-                Add First Item
-              </button>
-            </div>
-          )}
-      <form>
-        <ul>
-          {shoppingList &&
-            shoppingList[0] &&
-            shoppingList[0].items.map((shoppingItemObject, index) => {
-              return (
-                <li key={shoppingItemObject.shoppingListItemName + index}>
-                  <input
-                    type="checkbox"
-                    id={shoppingItemObject.shoppingListItemName}
-                    onChange={() => markItemAsPurchased(index)}
-                    checked={wasItemPurchasedWithinLastOneDay(
-                      shoppingItemObject.lastPurchasedOn,
-                    )}
-                  />
-                  <label htmlFor={shoppingItemObject.shoppingListItemName}>
-                    {shoppingItemObject.shoppingListItemName}
-                  </label>
-                </li>
-              );
-            })}
-        </ul>
-      </form>
+      {listHasNoItems && (
+        <div className="add-item">
+          <p>You haven't created a shopping list yet...</p>
+          <button type="submit" onClick={handleRedirect}>
+            Add First Item
+          </button>
+        </div>
+      )}
+      {listHasAtLeastOneItem ? (
+        <div className="search-list">
+          <SearchBar searchTerm={searchTerm} setSearchTerm={setSearchTerm} />
```

Comment:
Instead of passing the setState lets pass a function as a props that set's the state such as
`<SearchBar
  value={searchTerm}
  setValue={(searchTerm) => {
    setSearchTerm(searchTerm);
  }}
/>`
This avoids the strong coupling of parent's state with the child component.